src/main/java/core/basesyntax/dao/jdbc/DaoJdbcManufacturerImpl.java

```diff
@@ -0,0 +1,126 @@
+package core.basesyntax.dao.jdbc;
+
+import core.basesyntax.dao.DaoManufacturer;
+import core.basesyntax.exceptions.DataProcessingException;
+import core.basesyntax.lib.Dao;
+import core.basesyntax.model.Manufacturer;
+import core.basesyntax.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class DaoJdbcManufacturerImpl implements DaoManufacturer {
+    @Override
+    public Manufacturer create(Manufacturer item) {
+        String query = "INSERT INTO manufacturers (manufacturer_name, manufacturer_country) "
+                + "VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement
+                    = connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS);
```

Comment:
```suggestion
        try (Connection connection = ConnectionUtil.getConnection();
            PreparedStatement statement
                    = connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
```
and then we can remove `statement.close()`